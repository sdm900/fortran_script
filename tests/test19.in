print besjn(1,0-1)
print besjn(1,-1)