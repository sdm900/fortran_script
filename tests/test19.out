    -0.44005 
    -0.44005 
